packages/api/src/constants.ts

```diff
@@ -39,3 +39,8 @@ export const INCUBATION_POINTS_OTHERS = process.env.INCUBATION_POINTS_OTHERS
 export const MINT_PRIVATE_KEY =
   process.env.MINT_PRIVATE_KEY ||
   '0xb5b1870957d373ef0eeffecc6e4812c0fd08f554b37b233526acc331bf1544f7'
+
+// Number of different egg colors
+export const EGG_COLORS_LENGTH = process.env.EGG_COLORS_LENGTH
```

Comment:
```suggestion
export const EGG_COLORS_COUNT = process.env.EGG_COLORS_COUNT
```

Reply:
Silly thing, this tells me more clearly that this value represents how many colors exist.